npm/pkg/controlplane/translation/translatePolicy.go

```diff
@@ -34,6 +35,8 @@ var (
 	)
 	// ErrUnsupportedIPAddress is returned when an unsupported IP address, such as IPV6, is used
 	ErrUnsupportedIPAddress = errors.New("unsupported IP address")
+	// ErrUnsupportedNonCIDR is returned when non-CIDR blocks (i.e pod selectors or namespace selectors) are passed with a NPM Lite configuration
+	ErrUnsupportedNonCIDR = errors.New("unsupported Non-CIDR block for NPM Lite")
```

Comment:
we're actually allowing default allow and default deny policies as well (in `TranslatePolicy()`). Could we modify this error description to express that?

Reply:
Updated the description thanks

Reply:
nit: could we modify the error description as well to help a user in case they try debugging this error in logs?

I believe this Lite mode will actually work for all Policies except those with an ingress/egress pod selector or namespace selector

Reply:
updated the description

Reply:
sorry, I mean inside the code 😅

Reply:
haha got it! I updated it, @huntergregory let me know how that sounds

Reply:
Also great point out for named ports, added logic to output an error in case of named ports as well